### DISCUSSION TITLE
Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as ********"

### DISCUSSION THREAD

**Discussion by SamScholefield-open200** [community/community#110298] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

This error message is now being shown to me when using the GitHub CoPIlot extension in VSCODE (Version: 1.86.2 (Universal)). I am not logged into GitHub anywhere on this machine with the username it provides and running `git config --global user.name` does not return the username it is showing. Clicking on the extension icon in the tray and then clicking sign in throws an error. 

Please advise  how to reauthenticate the CoPilot extension with the correct account details? Happy to provide further info if required.

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
